opentelemetry-stdout/src/metrics/exporter.rs

```diff
@@ -163,16 +163,6 @@ fn print_histogram<T: Debug>(histogram: &data::Histogram<T>) {
 fn print_data_points<T: Debug>(data_points: &[data::DataPoint<T>]) {
     for (i, data_point) in data_points.iter().enumerate() {
         println!("\t\tDataPoint #{}", i);
-        let datetime: DateTime<Utc> = data_point.start_time.into();
```

Comment:
lets keep printing the start and endtime and not remove it.

Reply:
moved StartTime and EndTime printing outside of DataPoint, to reflect physical structure.